src/panels/scene-viewer/SceneViewer.tsx

```diff
@@ -159,11 +173,32 @@ export const SceneViewer = (props: SceneViewerPropsFromParent) => {
       dataBindingTemplate[undecoratedKey] = selectedPropertyValue;
     }
 
-    const selectedDataBinding = {
-      [DataBindingLabelKeys.entityId]: selectedEntityValue ?? '',
-      [DataBindingLabelKeys.componentName]: selectedComponentValue ?? '',
-      [DataBindingLabelKeys.propertyName]: selectedPropertyValue ?? '',
-    };
+    let selectedDataBinding: Record<string, string> | undefined =
+      selectedEntityValue && selectedComponentValue
+        ? {
+            [DataBindingLabelKeys.entityId]: selectedEntityValue,
+            [DataBindingLabelKeys.componentName]: selectedComponentValue,
```

Comment:
Why don't we set the selected propertyName?

Reply:
we did not use it to match the data binding, so I just removed it